src/main/java/Task.java

```diff
@@ -0,0 +1,21 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description){
```

Comment:
Perhaps you can consider leaving a white space after ) and before {. Same for the other methods in this file.